scripts/amd/gemm/matmul.py

```diff
@@ -1,166 +1,334 @@
-#!/usr/bin/env python3
-import argparse
-import sys
+"""
+Matrix Multiplication Tuning Scripts, Changed from the tutorial example "python/tutorials/03-matrix-multiplication.py"
+"""
 
-import pytest
 import torch
-from torch.testing import assert_close
 
 import triton
 import triton.language as tl
+import argparse
+import sys
 import yaml
+import os
+
+# global flag to indicate whether using the full tuing space
+tuning_full_space = False
```

Comment:
Can we make this an option of the tuning script?

Reply:
I tried using the global variable for that, but it does not work. Seems it is too late to read from input options